doc/source/serve/index.md

```diff
@@ -35,12 +35,9 @@ api/index
 
 (rayserve-overview)=
 
-Ray Serve is a scalable model serving library for building online inference APIs.
-Serve is framework-agnostic, so you can use a single toolkit to serve everything from deep learning models built with frameworks like PyTorch, TensorFlow, and Keras, to Scikit-Learn models, to arbitrary Python business logic. It has several features and performance optimizations for serving Large Language Models such as response streaming, dynamic request batching, multi-node/multi-GPU serving, etc.
+Ray Serve is a scalable, framework-agnostic model serving library for building online inference APIs. Serve integrates with any ML framework including PyTorch, TensorFlow, Keras, Scikit-Learn, and more. It's particularly well suited for model composition and many model serving, and includes performance optimizations for serving LLMs such as response streaming, dynamic request batching, multi-mode/multi-GPU serving, and more.
```

Comment:
Could you add a link to the first occurence of `Ray Serve` that goes to where the marketing content for Serve moved to?

Reply:
let's not forget this piece - "to arbitrary Python business logic" -> that's an important highlight. Also model composition and many model serving are more differentiated than the LLM serving features